doc/ArchInstall/arch_install.sh

```diff
@@ -29,8 +29,9 @@ mount ${ROOT} /mnt
 mkdir -p /mnt/boot/efi
 mount ${BOOT} /mnt/boot/efi
 
-# Bootstrap Pacman
+# Bootstrap Pacman, and trust everything to prevent pgp key errors
 pacstrap /mnt base linux linux-firmware
+cp /etc/pacman.conf /mnt/etc/pacman.conf
```

Comment:
You can do the `pacman.conf` with `SystemConfiguration` too. Copying `pacman.conf` is just creating another copy of the file that you need to rememebr to maintain.

Reply:
One thing you do need to do is make sure that when you're installing your mirror list for pacstrap is the same as the one on docker, otherwise you need to downgrade later

Reply:
That makes sense. But won't using that old mirrorlist likely give us expired pgp key errors now that we're not using the [pacman.conf ](https://github.com/NUbots/NUbots/pull/1174#discussion_r1295756966) file until systemconfigurator copies it over?

Reply:
It will, you'll need to make sure at least that setting is set in the conf file

Reply:
You can always make your install script download [pacman.conf](https://raw.githubusercontent.com/NUbots/NUbots/main/docker/etc/pacman.conf) from the repository. 

You could also make a similar `mirrorlist` file for docker (and update [Dockerfile](https://github.com/NUbots/NUbots/blob/main/docker/Dockerfile#L20) to use it). This will make it easier for you to keep to the docker image and the install scripts in sync